pubspec.yaml

```diff
@@ -39,7 +39,9 @@ dependencies:
   latlong2: ^0.9.0
   quickalert: ^1.0.1
   mocktail: ^1.0.0
-
+  flutter_map_location_marker: any
```

Comment:
why any?

Reply:
I guess I was following the readme. I will change that to specific version